README.md

```diff
@@ -85,6 +85,7 @@ If you want to benchmark Java programs, you need to set the
 set several other flags:
 
 ```sh
+$ make clean # if make has already been called
```

Comment:
For clarity, how about "To ensure rebuild if you have already built without Java."